archway-1/gentx/gentx-24cea010f283e8ca3285dab19b643b9b5ac78c61.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"B-Harvest","identity":"8957C5091FBF4192","website":"https://bharvest.io","security_contact":"contact@bharvest.io","details":"Provides secure validation services for dPoS networks"},"commission":{"rate":"0.100000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"archway1wr8v2eept0n26nsa706k5a9kssnw7kdajd0euu","validator_address":"archwayvaloper1wr8v2eept0n26nsa706k5a9kssnw7kdajfxm34","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"+xmVdMqVjqrBCPIX+IuzCCPP/5N6l1zt1xtsIDQ8364="},"value":{"denom":"aarch","amount":"1000000000"}}],"memo":"24cea010f283e8ca3285dab19b643b9b5ac78c61@155.55.88.0:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AwK1jZTou74FXe+GoHsNF7meefkBheIC8lfNvhmjrjbW"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[{"denom":"aarch","amount":"180000000000000000"}],"gas_limit":"200000","payer":"","granter":""}},"signatures":["vmAc5lhcOrONn2QLpl4IQowATagGIkqVDtB110hw974q83jSHOtiGvZKFSYYlEC1QiSqc5tH3bgtHlZ6eI/2SQ=="]}
```

Comment:
Was gentx generated with proper chain id? it should be `--chain-id archway-1`

Reply:
On our server:
```bash
cat genesis.json | jq .chain_id
"archway-1
```

```bash
archwayd validate-genesis
File at /home/ubuntu/.archway/config/genesis.json is a valid genesis file
```

If it's not valid, could we do again? 
1. re-send google form
2. re-do gentx PR

Reply:
I'll do it again asap.